tests/DucksNet.UnitTests/MedicalRecordTests.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DucksNet.UnitTests;
+public class MedicalRecordTests
+{
+    [Fact]
+    public void When_CreateMedicalRecord_Then_ShouldReturnMedicalRecord()
+    {
+        //ToDo: i don't think it is needed here ?
```

Comment:
Please maintain convetion for TODOs. Also please don't commit empty files unless the stubs are necessary for build to succeed.